ducktape/cluster/remoteaccount.py

```diff
@@ -153,3 +153,12 @@ def __str__(self):
             r += self.user + "@"
         r += self.hostname
         return r
+
+
+def http_request(url, method, data="", headers=None):
```

Comment:
Not crazy about this being here. Doesn't look like it has anything to do with the remote account functionality? Could we put it in a different utility module instead? Or better yet, maybe start a mixin class for HTTP/REST utilities since we'll probably need a bunch of them, and this can form the core?